README.md

```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Overview
+* Introduction
+  * Name: Boseong Yun (Peter)
+  * Major: Public Pollicy
+  * Acadmeic interests: gender equality
+  * hobby: Youtube
+
+
+* Reflections
+  * Github Workflow
+  * Rmarkdown
+  * Final Thoughts
+  
+  
+## Introduction
+![me](Michigan.jpg)
+
+My name is **Boseong Yun** and please feel free to call me **Peter**! I am a first-year grdaute student studying public policy at the Harris School of Public Policy. I am studying to quantify gender inequalites in text, image, sound, and other kinds of unstructured data. Yes, I have a very long way to go. For more information about me, you can check out my [Youtube channel](https://www.youtube.com/channel/UCfv5YsyMCcQPNyT-g0cEYGA) which might be more boring than our new lives quarantined at home. I will try to be funny...! Also, please do not hesitate to talk to me if you have any questions that I can help you with and it is really nice to meet you all! 
+
+## Reflections
```

Comment:
For future assignments, you can leave the reflection in the pull request text box. Not a big deal at all, but it usually satisfying to just see a clean report.

Reply:
Sure thing! I will leave the reflection part in the pull request text box. Thank you, deblnia!